packages/xo-web/src/xo-app/logs/backup-ng/index.js

```diff
@@ -49,10 +52,41 @@ export const LogStatus = ({ log, tooltip = _('logDisplayDetails') }) => {
   )
 }
 
+const GoToJob = decorate([
+  withRouter,
+  provideState({
+    effects: {
+      goTo() {
+        const { jobId, location, router, scrollIntoJobs } = this.props
+        router.replace({
+          ...location,
+          query: { ...location.query, s: `id:${jobId}` },
+        })
+        scrollIntoJobs()
+      },
+    },
+  }),
+  injectState,
+  ({ effects, children }) => (
+    <Tooltip content={_('goToThisJob')}>
+      <p onClick={effects.goTo}>{children}</p>
```

Comment:
Don't you want to add a `cursor: pointer` on this?

Reply:
The `cursor: pointer` was remove by this commit f03965c0819c653d2a8b883121240348e9f7d82b which is resultant to our discussion about letting the navigator decide in which cursor will be adopted in each case.